assets/js/controllers/transactions.controller.js

```diff
@@ -74,7 +74,11 @@ function TransactionsCtrl ($scope, $q, $translate, $uibModal, Wallet, MyWallet,
   $scope.$on('$destroy', unsub);
 
   // Searching and filtering
-  $scope.filterTypes = ['ALL', 'SENT', 'RECEIVED', 'TRANSFERRED'];
+  if (window.innerWidth <= 768) {
```

Comment:
Should really avoid dealing with layout logic in controllers, is there any way you could do this with css? If not (or if it's too much of a pain), could you use `$scope.$root.size.xs` instead?